src/bin/bat/main.rs

```diff
@@ -56,15 +59,43 @@ fn run_cache_subcommand(matches: &clap::ArgMatches) -> Result<()> {
     Ok(())
 }
 
+fn get_syntax_mapping_to_paths(
+    mappings: Vec<(GlobMatcher, MappingTarget)>,
+) -> HashMap<String, Vec<String>> {
+    let mut map = HashMap::new();
+    for mapping in mappings {
+        match mapping.1 {
```

Comment:
instead of matching on `mapping.1`, you could also match on `mapping` and then capture the tuple:
``` rust
match mapping {
    (_, MappingTarget::MapToUnknown) => {}
    (matcher, MappingTarget::MapTo(s)) => {}
}
```
This way, we don't have to use anonymous names like `.0` and `.1`.